[2023-05-02T08:46:48.009+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: prueba_tecnica.transform_data manual__2023-05-02T08:46:35.944044+00:00 map_index=0 [queued]>
[2023-05-02T08:46:48.023+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: prueba_tecnica.transform_data manual__2023-05-02T08:46:35.944044+00:00 map_index=0 [queued]>
[2023-05-02T08:46:48.023+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T08:46:48.024+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-05-02T08:46:48.024+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T08:46:48.040+0000] {taskinstance.py:1304} INFO - Executing <Mapped(_PythonDecoratedOperator): transform_data> on 2023-05-02 08:46:35.944044+00:00
[2023-05-02T08:46:48.047+0000] {standard_task_runner.py:55} INFO - Started process 641 to run task
[2023-05-02T08:46:48.053+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'prueba_tecnica', 'transform_data', 'manual__2023-05-02T08:46:35.944044+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/data_process/prueba/Prueba_Tecnica.py', '--cfg-path', '/tmp/tmp3b3q3yav', '--map-index', '0']
[2023-05-02T08:46:48.054+0000] {standard_task_runner.py:83} INFO - Job 36: Subtask transform_data
[2023-05-02T08:46:48.120+0000] {task_command.py:389} INFO - Running <TaskInstance: prueba_tecnica.transform_data manual__2023-05-02T08:46:35.944044+00:00 map_index=0 [running]> on host fc43d9cc2362
[2023-05-02T08:46:48.187+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=loliveros
AIRFLOW_CTX_DAG_ID=prueba_tecnica
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T08:46:35.944044+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-02T08:46:35.944044+00:00
[2023-05-02T08:46:48.188+0000] {Prueba_Tecnica.py:105} INFO - File path: /tmp/prueba_tecnica/data/raw/collisions_20230502.csv and file path dest: /tmp/prueba_tecnica/data/stage
[2023-05-02T08:46:49.559+0000] {warnings.py:109} WARNING - /opt/***/dags/data_process/prueba/utils/general_utils.py:24: DtypeWarning: Columns (27) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(file_path, sep = sep, encoding = encoding, header = header)

[2023-05-02T08:46:49.958+0000] {Prueba_Tecnica.py:119} INFO - Files downloaded: ['/tmp/prueba_tecnica/data/stage/collisions_20230502.csv']
[2023-05-02T08:46:49.988+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-02T08:46:49.998+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=prueba_tecnica, task_id=transform_data, map_index=0, execution_date=20230502T084635, start_date=20230502T084648, end_date=20230502T084649
[2023-05-02T08:46:50.072+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-05-02T08:46:50.128+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
